README.md

```diff
@@ -1,21 +1,30 @@
 # verify_ap_hits2015
-Data from HiTS (2015) to verify the performance of alert production in the LSST DM stack.
 
-This repo is designed to be used as a template for developing new data sets for integration into `verify_ap`.
+Data from HiTS (2015) to verify the performance of alert production in the LSST DM stack.
 
-It is currently set up for using `obs_test`.
+Contains DeCam data from the HiTS (2015) fields `Blind15A_26`, `Blind15A_40`, and `Blind15A_42`.
 
 Relevant Files and Directories
 -----
 path                  | description
 :---------------------|:-----------------------------
-`raw`                 | To be populated with photometrically and astrometrically calibrated data. Currently contains a single small fits file (taken from `obs_test`) to test `git-lfs` functionality.
-`calib`               | To be populated with master calibs. Currently empty.
+`raw`                 | Photometrically and astrometrically calibrated compressed DeCam fits images from the HiTS (2015) fields `Blind15A_26`, `Blind15A_40`, and `Blind15A_42`.
+`calib`               | DeCam master calibs and `wtmap`s from the 2015 HiTS campaign. No raw images. See below for filename information.
```

Comment:
Probably not, although I suppose some of the details from the main README could go here.